index.html

```diff
@@ -0,0 +1,41 @@
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title id="title">Dr. Norman Borlaug</title>
+        <link rel="stylesheet" href="css/style.css">
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/css/bootstrap.min.css">
+      </head>
+
+
+<body class="text-center">
+    <div id="main" >
+        <h1>Dr. Norman Borlaug</h1>
+        <p style=font-size:15px> The man who saved a billion lives
+            </p>
+     <div id="img-div">
+         <img  id="image"
+         src="https://c2.staticflickr.com/4/3689/10613180113_fdf7bcd316_b.jpg">
+         <p> 
+                Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.
+         </p>
+     </div>       
+     <div id ="tribute-info"></div>
+     <p><b>Here's a time line of Dr. Borlaug's life:</b></p>
+     <ul>
+            <li>1914 - Born in Cresco, Iowa</li>
+            <li>1933 - Leaves his family's farm to attend the University of Minnesota, thanks to a Depression era program known as the "National Youth Administration"</li>
+            <li>1935 - Has to stop school and save up more money. Works in the Civilian Conservation Corps, helping starving Americans. "I saw how food changed them", he said. "All of this left scars on me."</li>
+            <li>1937 - Finishes university and takes a job in the US Forestry Service</li>
+            <li>1938 - Marries wife of 69 years Margret Gibson. Gets laid off due to budget cuts. Inspired by Elvin Charles Stakman, he returns to school study under Stakman, who teaches him about breeding pest-resistent plants.</li>
+            <li>1941 - Tries to enroll in the military after the Pearl Harbor attack, but is rejected. Instead, the military asked his lab to work on waterproof glue, DDT to control malaria, disinfectants, and other applied science.</li>
+        
+        </ul>
+        
+     </div>
+
+
+    </div>
```

Comment:
- [] [REQUIRED] Let's make sure not to have extra closing tags. This way it will be more professional.